### PR TITLE
test: revert quality to 224x224

### PR DESCRIPTION
**Pull request by florian-hoenicke**
- [x] deployment is 2x to 4x faster
- [ ] quality needs to be checked

### CODE REVIEW

File: tests/executor/preprocessor/test_unit.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def download_mock(url, destfile):
         (
             'image/6785325056.jpg',
             1,
-            'Richard Branson TotallyLooksLike.com Zaphod Beeblebrox',
+            'RichavdE',  # with larger resolution it would be the following text: 'Richard Branson TotallyLooksLike.com Zaphod Beeblebrox',
```
Comment (makram93) [inline tests/executor/preprocessor/test_unit.py:26]: But isn't this harmful? Or are we just concerned with finding any text and not full text?

**Comment by joschkabraun** [comment]
Which queries did you try? Did you document the tested queries somewhere?

File: tests/executor/preprocessor/test_unit.py (hunk 1 above)
Comment (florian-hoenicke) [reply tests/executor/preprocessor/test_unit.py:26]: yes, just what text it is at the moment

**Comment by florian-hoenicke** [comment]
> Which queries did you try? Did you document the tested queries somewhere?

Just a few queries we had before. You are right, it is not statistically significant. "girl on a motorbike" and "campfire".
I do a side by side comparison before.

**Pull request merged**